openassessment/templates/openassessmentblock/response/oa_response_graded.html

```diff
@@ -26,9 +25,7 @@
     <div class="wrapper--step__content">
         <div class="step__content">
             <article class="submission__answer__display">
-                <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
-
-                {% trans "Your response to the question above:" as translated_label %}
+                {% trans "Your response" as translated_label %}
```

Comment:
Ohhhh, this is just defining the variable "translated_label" which is then passed into the template. Got it now!